### ISSUE TITLE
Bugs related to how event chains works

### ISSUE THREAD

**Issue by Lendemor** [pynecone-io/pynecone#317]
**Describe the bug**
Trying to use event chains, I ran into a couple problems: 

**To Reproduce**
Problems encountered:

1. `partial` work if I'm using a single event handler, but not with chain.
```
pc.button("Trigger", on_click=[partial(State.subevent1, var), State.subevent2])
```

2. In this case, the `var` that I passed to `sub_event1` is a `str`, but the value become `'"my value"'` instead of `"my value"`
```
# in State
def event(self, var):
        return [self.subevent1(var), self.subevent2]

# in render
pc.button("Trigger", on_click=partial(State.event, "my value")
```

**Expected behavior**
Chained Events trigger correctly 

** Specifics:**
 - Python Version: 3.10
 - Pynecone Version: 0.1.13

**Comment by LumiaGG** [comment]
I have an idea. I think we can fix him.
I'll submit a pr later.

**Comment by picklelo** [comment]
Closed by #413
